### ISSUE TITLE
Component error: pool_pump - Integration 'pool_pump' not found.

### ISSUE THREAD

**Issue by fabiofare** [oncleben31/ha-pool_pump#23]
HA Core Versione 2021.6.4
HA OS 5.12

## Version of the custom_component
VERSION = "0.0.1"

## Configuration

added to configuration.yaml
```
pool_pump:
  switch_entity_id: switch.pool_pump_switch
  pool_pump_mode_entity_id: input_select.pool_pump_mode
  pool_temperature_entity_id: input_number.pool_water_temperature
  # optional:
  water_level_critical_entity_id: binary_sensor.pool_water_level_critical
  schedule_break_in_hours: 1.0
```

added to a dedicated package/pool_pump.yaml, I currently do not have a water thermostat hence I create an input_number as suggested.
```
input_number:
  pool_water_temperature:
    name: Manual Water Temperature
    min: 15
    max: 45
    step: 0.5

input_select:
  pool_pump_mode:
    name: Pool Pump mode
    options:
      - 'Auto'
      - 'On'
      - 'Off'
    initial: 'Auto'
    icon: mdi:water-pump

automation:
  - alias: 'Pool Pump On'
    trigger:
      - platform: state
        entity_id: input_select.pool_pump_mode
        to: 'On'
    action:
      service: homeassistant.turn_on
      entity_id: switch.pool_pump_switch

  - alias: 'Pool Pump Off'
    trigger:
      - platform: state
        entity_id: input_select.pool_pump_mode
        to: 'Off'
    action:
      service: homeassistant.turn_off
      entity_id: switch.pool_pump_switch

  - alias: 'Check Pool Pump Periodically'
    trigger:
      - platform: time_pattern
        minutes: '/5'
        seconds: 00
    action:
      service: pool_pump.check

  - alias: 'Check Pool Pump on Event'
    trigger:
      - platform: homeassistant
        event: start
      - platform: state
        entity_id:
          - input_number.pool_water_temperature
          - input_select.pool_pump_mode
          - binary_sensor.pool_water_level_critical
    action:
      service: pool_pump.check

```

HA Logs here.
```
2021-06-13 22:57:36 ERROR (MainThread) [homeassistant.components.hassio] The system cannot restart because the configuration is not valid: Component error: pool_pump - Integration 'pool_pump' not found.
2021-06-13 22:57:36 ERROR (MainThread) [aiohttp.server] Error handling request
Traceback (most recent call last):
File "/usr/local/lib/python3.8/site-packages/aiohttp/web_protocol.py", line 422, in _handle_request
resp = await self._request_handler(request)
File "/usr/local/lib/python3.8/site-packages/aiohttp/web_app.py", line 499, in _handle
resp = await handler(request)
File "/usr/local/lib/python3.8/site-packages/aiohttp/web_middlewares.py", line 119, in impl
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/security_filter.py", line 60, in security_filter_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/forwarded.py", line 76, in forwarded_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/request_context.py", line 24, in request_context_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/ban.py", line 78, in ban_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/auth.py", line 144, in auth_middleware
return await handler(request)
File "/usr/src/homeassistant/homeassistant/components/http/view.py", line 135, in handle
result = await result
File "/usr/src/homeassistant/homeassistant/components/api/__init__.py", line 376, in post
await hass.services.async_call(
File "/usr/src/homeassistant/homeassistant/core.py", line 1491, in async_call
task.result()
File "/usr/src/homeassistant/homeassistant/core.py", line 1526, in _execute_service
await handler.job.target(service_call)
File "/usr/src/homeassistant/homeassistant/components/hassio/__init__.py", line 518, in async_handle_core_service
raise HomeAssistantError(
homeassistant.exceptions.HomeAssistantError: The system cannot restart because the configuration is not valid: Component error: pool_pump - Integration 'pool_pump' not found.
```

## Describe the bug
```
The custom component is not loaded

**Comment by FraZe69** [comment]
I was helped me to add in the:

/config/custom_components/pool_pump/manifest.json

the following line:

 "version": "1.0",

**Comment by oncleben31** [comment]
Similar to #16 and fixed by #20
